app/views/assessments/_edit_problems.html.erb

```diff
@@ -26,3 +47,7 @@
 </table>
 <br>
 <%= link_to "Add Problem".html_safe, new_course_assessment_problem_path(@course, @assessment), { :class=> "btn" } %>
+
+<div class="action_buttons">
```

Comment:
Saving brings us back to the basic tab. It seems that the `tab_index` method in `assessments_controller.rb` is missing the check for the problems tab.

i.e., we need to add
`elsif params[:problems]`
`tab_name = "problems"`

Reply:
let me look into that

Reply:
fixed that